dev-docs/bidders/adnuntius.md

```diff
@@ -5,29 +5,26 @@ description: Prebid Adnuntius Bidder Adaptor
 pbjs: true
 biddercode: adnuntius
 media_types: banner
-gdpr_supported: false
+gdpr_supported: true
 ---
 
-
 ### Bid Params
 
 {: .table .table-bordered .table-striped }
-| Name        | Scope    | Description                                                          | Example  | Type     |
+| Name | Scope | Description | Example | Type |
 |-------------|----------|----------------------------------------------------------------------|----------|----------|
-| `auId`      | required | The ad unit ID `'0000000000072345'` leading zeros can be omitted.    | `string` |          |
-| `network`   | optional | Used if you want to make requests to multiple networks in adnuntius. | `string` |          |
-| `targeting` | optional | Targeting to be sent through to adnuntius with the request.          | `string` |          |
-
-
+| `auId` | required | The ad unit ID `'0000000000072345'` leading zeros can be omitted. | `string` | |
```

Comment:
`string` should go in the Type column. The Example column should have an example of a string that would be assigned to that param.